aether-kernel/aether/kernel/api/tests/test_serializers.py

```diff
@@ -358,10 +363,9 @@ def test__serializers__create_and_update(self):
         with self.assertRaises(ValidationError) as ve:
             entity_7.save()
         self.assertIn(
-            'No associated project. Check you provided the correct Submission, Mapping and Schema Decorator',
+            'Schema Decorator MUST be provided with entities',
             str(ve.exception)
         )
```

Comment:
Same comment as above. Why was it wrong? :thinking:

Reply:
Bad paste. Surprised the test passed like this. :man_shrugging: